test/cuda/test_execution_engine_db.cc

```diff
@@ -21,18 +21,17 @@
 #include <glog/logging.h>
 #include <gtest/gtest.h>
 
```

Comment:
execution_engine does not exist anymore, rename/merge this test?

Reply:
killing it too